docs/contributor/releasing.md

```diff
@@ -19,66 +22,56 @@ This release process covers the steps to release new major and minor versions fo
+   git tag {RELEASE_VERSION}
    ```
 
-9. Push the tag to the upstream repository.
+8. Push the tag to the upstream repository.
 
    ```bash
-   git push {REPOSITORY_REMOTE} v{RELEASE_VERSION}
+   git push {REPOSITORY_REMOTE} {RELEASE_VERSION}
    ```
 
-   The {RELEASE_VERSION} tag triggers a GitHub action (`GitHub Release`). 
+   The {RELEASE_VERSION} tag triggers a GitHub action (`GitHub Release`).
 
-10. Verify the [status](https://github.com/kyma-project/opentelemetry-collector-components/actions) of the GitHub action (`GitHub Release`).
-    - After the GitHub action succeeded, the new GitHub release is available under [releases](https://github.com/kyma-project/opentelemetry-collector-components/releases).
-    - If the GitHub action fails, re-trigger it by removing the {RELEASE_VERSION} tag from upstream and pushing it again:
+9. Verify the [status](https://github.com/kyma-project/opentelemetry-collector-components/actions) of the GitHub
+   action (`GitHub Release`).
+   - After the GitHub action succeeded, the new GitHub release is available
+     under [releases](https://github.com/kyma-project/opentelemetry-collector-components/releases).
+   - If the GitHub action fails, re-trigger it by removing the {RELEASE_VERSION} tag from upstream and pushing it
+     again:
 
-      ```bash
-      git push --delete upstream v{RELEASE_VERSION}
-      git push upstream v{RELEASE_VERSION}
-      ```
+     ```bash
+     git push --delete upstream v{RELEASE_VERSION}
+     git push upstream v{RELEASE_VERSION}
+     ```
 
-11. If the previous release was a bugfix version (patch release) that contains cherry-picked changes, these changes might appear again in the generated change log. If there are redundant entries, edit the release description and remove them.
+10. If the previous release was a bugfix version (patch release) that contains cherry-picked changes, these changes
+    might appear again in the generated change log. If there are redundant entries, edit the release description and
+    remove them.
 
 ## Changelog
 
-Every PR's title must adhere to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification for an automatic changelog generation. It is enforced by a [semantic-pull-request](https://github.com/marketplace/actions/semantic-pull-request) GitHub Action.
+Every PR's title must adhere to the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification
+for an automatic changelog generation. It is enforced by
+a [semantic-pull-request](https://github.com/marketplace/actions/semantic-pull-request) GitHub Action.
 
 ### Pull Request Title
 
-Because of the squash-and-merge GitHub workflow, each PR results in a single commit after merging into the main development branch. The PR's title becomes the commit message and must adhere to the template:
+Because of the squash-and-merge GitHub workflow, each PR results in a single commit after merging into the main
```

Comment:
```suggestion
Because of the squash-and-merge GitHub workflow, each PR results in a single commit after merging into the main development branch. The PR's title becomes the commit message and must adhere to the template:
```